packages/manager/cypress/e2e/core/longview/longview.spec.ts

```diff
@@ -176,4 +184,44 @@ describe('longview', () => {
         });
     });
   });
+
+  /*
+   * - Confirms that the landing page empty state message is displayed when no Longview clients are present.
+   * - Confirms that UI updates to show the new client when creating one.
+   */
+  it('displays empty state message when no clients are present and shows the new client when creating one', () => {
+    const client: LongviewClient = longviewClientFactory.build();
+    const status: LongviewResponse = longviewResponseFactory.build();
+    mockGetLongviewClients([]).as('getLongviewClients');
+    mockCreateLongviewClient(client).as('ceateLongviewClient');
+    mockFetchLongviewStatus(status).as('fetchLongviewStatus');
+
+    cy.visitWithLogin('/longview');
+    cy.wait('@getLongviewClients');
+
+    // Confirms that a landing page empty state message is displayed
+    cy.findByText(longviewEmptyStateMessage).should('be.visible');
+    cy.findByText(longviewAddClientButtonText).should('be.visible');
+
+    ui.button
+      .findByTitle(longviewAddClientButtonText)
+      .should('be.visible')
+      .should('be.enabled')
+      .click();
+    cy.wait('@ceateLongviewClient');
+
+    // Confirms that UI updates to show the new client when creating one.
+    cy.findByText(`longview-client-${client.id}`).should('be.visible');
```

Comment:
```suggestion
    cy.findByText(`${client.label}`).should('be.visible');
```
Very minor change, but this way if the Longview client factory label ever changes the test will still pass 👍